src/integrationTest/java/com/mongodb/hibernate/BasicInsertionIntegrationTests.java

```diff
@@ -48,7 +48,7 @@ class BasicInsertionIntegrationTests {
     private static MongoConfiguration config;
 
     @BeforeEach
-    void setUp(SessionFactoryScope scope) {
+    void beforeEach(SessionFactoryScope scope) {
```

Comment:
We call `beforeAll` the methods annotated with `@BeforeAll`, and we call `afterAll` the methods annotated with `@AfterAll`. For consistency, it makes sense to call `beforeEach` the methods annotated with `@BeforeEach`, and call `afterEach` the methods annotated with `@AfterEach`.

Reply:
👍  It does make sense.